### ISSUE TITLE
email benachrichtigungstext

### ISSUE THREAD

**Issue by tickel69** [tkrebs/ep3-bs#169]
ich habe mir das ganze für'ne ferienwohung umgebaut, funz. auch echt super ... nur der text
in der bestätigungs email passt nicht, kann ich da auch nur von bis datum ohne uhrzeit anzeigen lassen, geht wahrscheinlich nur manuell ... ich finde das skript aber nicht ... Danke :)

**Comment by tkrebs** [comment]
Ja, für die E-Mails müssen wir bald mal ein über das Backend bearbeitbares Template-System einführen.

Bis dahin ist der E-Mail-Text der Buchungsbestätigung hier hinterlegt:

https://github.com/tkrebs/ep3-bs/blob/master/module/Booking/src/Booking/Service/Listener/NotificationListener.php#L76

**Comment by tickel69** [comment]
Danke für die schnelle Antwort :) Ich blick da nicht so ganz durch ... mit den vielen Variablen, wenn ich von Datum bis Datum ausgeben möchte wo und was muss ich ändern ? Danke :)

**Comment by MarcusThiel** [comment]
> Ja, für die E-Mails müssen wir bald mal ein über das Backend bearbeitbares Template-System einführen.
> 
> Bis dahin ist der E-Mail-Text der Buchungsbestätigung hier hinterlegt:
> 
> https://github.com/tkrebs/ep3-bs/blob/master/module/Booking/src/Booking/Service/Listener/NotificationListener.php#L76

Hallo, wenn ich den Text ändere, der ja englisch ist, wird er dann automatisch übersetzt in Deutsch ?

**Comment by MarcusThiel** [comment]
> Hallo, wenn ich den Text ändere, der ja englisch ist, wird er dann automatisch übersetzt in Deutsch ?

habe gerade gesehen das es schon einen Post dazu gibt ... mir wurde geholfen !!

https://github.com/tkrebs/ep3-bs/issues/153

**Comment by MarcusThiel** [comment]
Allerdings ... wo genau finde ich die deutsche Übersetzung zu dem englischen Benachrichtigungstext zur Buchung, irgendwo hier denke ich 

https://github.com/tkrebs/ep3-bs/tree/master/data/res/i18n/de-DE

aber in welcher Datei genau ?
